python/vyos/ifconfig/interface.py

```diff
@@ -1535,7 +1535,7 @@ def update(self, config):
         # before mangling any IPv6 option. If MTU is less then 1280 IPv6 will be
         # automatically disabled by the kernel. Also MTU must be increased before
         # configuring any IPv6 address on the interface.
-        if 'mtu' in config:
+        if 'mtu' in config and jmespath.search('dhcp_options.mtu', config) == None:
```

Comment:
Please use `dict_search()` instead - should be faster.

Reply:
> Please use `dict_search()` instead - should be faster.

Done!